tests/terraform/parser/resources/parser_scenarios/ternaries/main.tf

```diff
@@ -6,6 +6,11 @@ locals {
   bool_true  = true ? "correct" : "wrong"
   bool_false = false ? "wrong" : "correct"
 
+//  local_true = true
+  // TODO: HCL2 parser doesn't like the following line
+//  multiline = (local.local_true) ?
+//    "correct" : "wrong"
```

Comment:
@robeden I see it is commented out, so I'm guessing we don't support it? If not I'd like to have an issue on it

Reply:
Filed #864. Note that #863 is also relevant to this PR.